plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/DefaultQueryBuilder.java

```diff
@@ -340,6 +342,13 @@ protected String formatProjections(JdbcClient client, Map<JdbcColumnHandle, Stri
                 .collect(joining(", "));
     }
 
+    protected String formatProjectionAliases(JdbcClient client, String relationAlias, Map<JdbcColumnHandle, String> projections)
```

Comment:
given you don't need a Map, maybe take `Collection<String> projections`?

Reply:
Done!